components/Home/NodeFeatures/index.test.tsx

```diff
@@ -0,0 +1,14 @@
+import { render } from '@testing-library/react';
```

Comment:
tests should be inside `__tests__` as all other component do that

Reply:
Yes! Wait what I have done ...